localstack/services/sqs/provider.py

```diff
@@ -91,6 +92,19 @@
 # see https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_DeleteQueue.html
 RECENTLY_DELETED_TIMEOUT = 60
 
+IMMUTABLE_QUEUE_ATTRIBUTES = [
+    # these attributes cannot be changed by set_queue_attributes and should
+    # therefore be ignored when comparing queue attributes for create_queue
```

Comment:
Nit: maybe change the name. While I get what this is, "immutable" may cause the misconception that the values do not change. (It really depends on how you define "mutable" in this context I guess).

Reply:
Brainstorming: Maybe "INTERNAL_QUEUE_ATTRIBUTES"? with the "external" ones being the ones that can be changed from the outside via api call